gcloud-build-task/README.md

```diff
@@ -0,0 +1,3 @@
+﻿# gcloud-build-task
+
+A Task to run an arbitrary gcloud command.
```

Comment:
Nit, this could be a more descriptive text explaining the structure of the task, parameters etc...